tests/spread/integration/apache2-utils/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Integration tests for apache2-utils
+
+execute: |
+  # Installing apache2_bins to be able to do a benchmark test using ab util
+  rootfs="$(install-slices apache2-utils_bins apache2-utils_scripts apache2_bins base-passwd_data)"
```

Comment:
Instead of installing `apache2_bins`, can we install nginx to use as a webserver to test against instead?
That would help confirm that nothing from our apache2-utils actually requires components from the apache2 slices.